doc/source/myrocks/zenfs.rst

```diff
@@ -0,0 +1,95 @@
+.. _zenfs:
+
+=========================================
+RocksDB with ZenFS
+=========================================
+
+Implemented in Percona Server for MySQL 8.0.26-16.
+
+`ZenFS <https://zonedstorage.io/projects/zenfs/>`__ is a file system plugin which places files into zones on a raw zoned block device (ZBD) using the RocksDB File System interface. ZenFS depends on ``libzbd`` and requires a Linux kernel implementation which supports NVMe Zoned Namespaces. The kernel must be configured with `zone block device support enabled <https://zonedstorage.io/linux/config/#kernel-configuration>`__. 
+
```

Comment:
Could we add a note here about OS recommendations/advice? Given that support for ZenFS is still evolving, we should at least provide some general advice on how to get started, pointing to https://zonedstorage.io/distributions/linux/ for up to date information about OS support.

Reply:
Note added.

Reply:
Sine this is not a 'retail' product, one must go to WDC for all information  on OS and  kernel support.  Our instructions are specifically NOT get into environmental recommendations and configurations.  It is to be assumed that by the time a user gets to our software, the environment has been correctly built out and tested.  So the most we can/should do here is to refer elsewhere.  The problem with https://zonedstorage.io/distributions/linux/ is that it is not accurate and implies that MyRocks ZenFS will work on Hirsute and others out of the box and it will not due to the fact that all currently available kernels are missing required fixes and require re-compilation.  I do not expect that information to ever be properly represented on that page.